README.md

```diff
@@ -34,6 +35,9 @@ This plugin uses [OkHttp](http://square.github.io/okhttp/) to send a POST reques
 If you are using multiple GithubRelease tasks to compose a single release, the release must be published `draft=false` with the first task. Currently, the plugin cannot find an existing release if it is a draft.
 
 ## Changelog
+2.4.0

```

Comment:
📝 I'm not sure how strictly this repo follows semver, but as a new feature I thought a minor bump would be appropriate.